invenio_accounts/views.py

```diff
@@ -91,3 +91,14 @@ def init_menu():
             _("Change password"),
             order=0,
         )
+
+
+@blueprint.before_app_first_request
+def check_security_settings():
+    """Warn if session cookie is not secure in production."""
+    if (not current_app.debug and not current_app.testing and
```

Comment:
I was trying to understand the condition and make it more readable. What do you think about following code?

```python
in_production = not (current_app.debug or current_app.testing)
secure = current_app.config.get('SESSION_COOKIE_SECURE')
if in_production and not secure:
    current_app.logger.warning(...)
```